nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -360,6 +360,17 @@ SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "{{ secret_key }}")
 # if "NAUTOBOT_DEVICE_NAME_AS_NATURAL_KEY" in os.environ and os.environ["NAUTOBOT_DEVICE_NAME_AS_NATURAL_KEY"] != "":
 #     DEVICE_NAME_AS_NATURAL_KEY = is_truthy(os.environ["NAUTOBOT_DEVICE_NAME_AS_NATURAL_KEY"])
 
+# Event Brokers
+# EVENT_BROKERS = {
+#     "SyslogEventBroker": {
+#         "CLASS": "nautobot.core.events.SyslogEventBroker",
+#         "TOPICS": {
+#             "INCLUDE": ["nautobot.test.*"],
+#             "EXCLUDE": ["*.no-publish*"],
+#         },
+#     }
+# }
```

Comment:
Generally in this file the commented-out case should be the same as the default value for the config, so this would just be:

```suggestion
# EVENT_BROKERS = {}
```